peps/pep-0772.rst

```diff
@@ -0,0 +1,455 @@
+that PyPA should be expected to provide opinions, insight and experience when
+ecosystem-wide changes are being proposed. In particular, the work to support
+existing projects and maintain a user guide has been excellent.
+
+At the time of writing, PyPA is still less formal than its counterpart on the
+CPython side. It does not have a community elected body and individuals within
+the group meet ad hoc, rather than on a regular cadence. The PyPA itself is
+defined by a group of projects rather than as a group of individuals, with no
+direct oversight over its individuals.
+
+------------
+Packaging-WG
+------------
+
+The PSF's `Packaging Workgroup`_ was created to support the larger efforts of
+improving and maintaining the packaging ecosystem in Python through fundraising
+and disbursement of raised funds. It was expected to largely focus on efforts
+such as PyPI, pip, packaging.python.org, Setuptools, and cross-project efforts.
+
+The group made very good progress here in the past, but the scope and scale of
+supporting Python's packaging ecosystem has grown considerably in the last few
+years. There is both more work to be done and more stakeholders than ever
+before. The Packaging Workgroup is also not a community elected body and does
+not have a regular mechanism for changing, adding or removing members.
+
+--------------------------
+Interoperability Standards
+--------------------------
+
+The PyPA is responsible for creating and maintaining the interoperability
+standards for Python packaging tools. The decision making for updating these
+standards involves a standing delegation from the Python Steering Council to
+specific individuals based on specific topics, and these individuals have the
+ability to further delegate the decision making on the changes to other
+individuals.
+
+We know that this process is not sustainable and there has been a desire
+expressed by many to have a designated body to make these decisions (including
+the individuals with the standing delegations).
+
+-----------------------
+Python Steering Council
+-----------------------
+
+While there is overlap between the PyPA, Packaging-WG as well as the Python
+core team, the Steering Council is not well-placed to make decisions over
+Python packaging matters directly. Packaging only tangentially intersects with
+Python language evolution, packaging requires specialised expertise and deep
+domain knowledge (much like typing or the C API), and there is a different
+cohort of potential electors for a Packaging Council than the Steering Council.
+
+------------
+Expectations
+------------
+
+An elected Packaging Council that has authority over the interoperability
+standards and Python packaging tooling would have the mandate to coordinate
+efforts around Python packaging. This would provide a better, more consistent
+experience to packagers, package consumers, tool developers, PyPI, and other
```

Comment:
I think this is the first time PyPI is used?
```suggestion
experience to packagers, package consumers, tool developers, the Python Package Index (PyPI), and other
```

Reply:
It's earlier at line 64, while I'll update to call out this acronym like this!